.github/workflows/analyze.yml

```diff
@@ -0,0 +1,33 @@
+name: Analyze
+on:
+  pull_request:
+  push:
+    branches:
+      - confluence
+      - next
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      security-events: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # Needed for commands that depend on git tags
+          fetch-depth: 0
+      - name: Install dependencies
```

Comment:
you should perhaps use setup-node action here, to keep the version consistent. I think the default github runner comes with node-12 or node-14.

https://github.com/livepeer/livepeer-com/blob/master/.github/workflows/build.yaml#L24-L27

```yaml
      - uses: actions/setup-node@v3
        with:
          node-version: "16"
          cache: "yarn"
```